src/shared/clients/codecatalystClient.ts

```diff
@@ -411,12 +417,28 @@ class CodeCatalystClientInternal {
 
     /**
      * Gets a flat list of all repos for the given CodeCatalyst user.
+     * @param firstPartyOnly Get only 1P repositories. 1P being repos that are
+     *                       created in CodeCatalyst.
      */
     public listSourceRepositories(
-        request: CodeCatalyst.ListSourceRepositoriesRequest
+        request: CodeCatalyst.ListSourceRepositoriesRequest,
+        firstPartyOnly: boolean = false
```

Comment:
nit: is "third party" a more common term and more discoverable term than "first party" ?

```suggestion
        thirdparty: boolean = true
```

Reply:
Makes sense to me, refactored